src/components/FoodBox.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+
+export default class FoodBox extends React.Component {
+    constructor(props){
+      super(props)
+      this.state={
+        food:{
+          name: this.props.food.name,
+          calories: this.props.food.calories,
+          quantity: 0
+        }
+      }
+    }
+
+  changeQuantity=(event)=>{
+    this.setState({food:{
+      name: this.props.food.name,
+      calories: this.props.food.calories,
+      quantity: event.currentTarget.value
+    }})
+  }
+
+  
+  render(){
+  return(
+    <div className="box">
+      <article className="media">
+        <div className="media-left">
+          <figure className="image is-64x64">
+            <img src={this.props.food.image} alt='' className='food-pic'/>
```

Comment:
if you have multiple variables passed from props or the state. It is good practice to destructure them before the return and then use the individual variables.